src/utils/isValidZipCode.js

```diff
@@ -0,0 +1,18 @@
+const ZIP_COUNT = 48073;
+
+/**
+ * Check if a given integer is a valid zip code (between 1000 and 48073)
+ * @param {Number} code zip code
```

Comment:
This function still work if we pass '01000' 

```suggestion
 * Check if a given parameter is a valid zip code (between 1000 and 48073)
 * @param {Number|String} code zip code
```